### PR TITLE
Explain how to release module native

### PR DESCRIPTION
**Pull request by matks**

### CODE REVIEW

**Comment by matks** [comment]
Fixes partially https://github.com/PrestaShop/docs/issues/341

File: src/content/1.7/release/release-a-native-PS-module.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+Commit the changes with git message:
+```
+// Version updated to v<new_version>
+```
+Get this PR approved and merge it into `dev`. See an [example](https://github.com/PrestaShop/ps_shoppingcart/pull/50/) of such PR. 
+
+### Merge dev into master
+
+Then create a new PR to merge branch `dev` into branch `master`. See an [example](https://github.com/PrestaShop/ps_shoppingcart/pull/51) of such PR.
```
Comment (Progi1984) [inline src/content/1.7/release/release-a-native-PS-module.md:62]:
```suggestion
Create a new PR to merge branch `dev` into branch `master`. See an [example](https://github.com/PrestaShop/ps_shoppingcart/pull/51) of such PR.
```

File: src/content/1.7/release/release-a-native-PS-module.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+If QA validates the PR, it can be merged, which will update the `master` branch with the new code changes from `dev`.
+
+### Create a GitHub Release
+
+If you have enabled Release Drafter, it will create a draft release for you. You can review it and publish it.
+
+Else, you need to do it manually:
+
+Create a new tag with "v<new_version>" that targets the last commit on `master` and push it. This will publish the module version on https://packagist.org/ .
+
```
Comment (Progi1984) [inline src/content/1.7/release/release-a-native-PS-module.md:75]:
```suggestion
```

Done with `Create a release`

Comment (matks) [reply src/content/1.7/release/release-a-native-PS-module.md:75]: No, it's 2 different things. Creating the git tag is an action inside git metadata while creating a release is creating a new item in GitHub database.

Maybe I can use better words to make this unambiguous ?

Comment (Progi1984) [reply src/content/1.7/release/release-a-native-PS-module.md:75]: Fake news ;) 

When you create a release on Github, that create a tag.

image

Comment (matks) [reply src/content/1.7/release/release-a-native-PS-module.md:75]: Ah, this is what you meant 😄 but this is still 2 different things. However GitHub is able to "kill 2 birds with 1 stone".

But I think it's better to explain the 2 things in 2 separate lines, just like here https://github.com/PrestaShop/docs/pull/593

This is a documentation, it aims to be as complete as possible 😉 it's better IMO to explain each thing separately so people can grasp what each step is about.

However I can update the PR to provide a tip "by the way, GitHub can do this for you with the option ..."

Comment (Progi1984) [reply src/content/1.7/release/release-a-native-PS-module.md:75]: Ok, for the last option ;)

Comment (matks) [reply src/content/1.7/release/release-a-native-PS-module.md:75]: Fixed in https://github.com/PrestaShop/docs/pull/591/commits/415b4cd1258516cc371bb750269e85a68546e04b

File: src/content/1.7/release/release-a-native-PS-module.md (hunk 1 above)
Comment (matks) [reply src/content/1.7/release/release-a-native-PS-module.md]: Fixed in https://github.com/PrestaShop/docs/pull/591/commits/415b4cd1258516cc371bb750269e85a68546e04b

**Comment by matks** [comment]
Thank you for review

**Pull request merged**